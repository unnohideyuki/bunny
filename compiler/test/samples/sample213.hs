main = do putStrLn $ (\_ -> "PWildcard * 1") 0
          putStrLn $ (\a _ -> show a ++ ", PWildcard") 0 1
          putStrLn $ (\_ b -> "PWildcard, " ++ show b) 0 1
          putStrLn $ (\_ _ -> "PWildcard * 2") 0 1


