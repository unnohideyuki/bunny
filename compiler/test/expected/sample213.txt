PWildcard * 1
0, PWildcard
PWildcard, 1
PWildcard * 2
